src/components/Notification/Icon.tsx

```diff
@@ -34,7 +34,7 @@ const NotificationIcon: FC = () => {
       }}
     >
       <LightningBoltIcon className="w-5 h-5 sm:w-6 sm:h-6" />
-      {showBadge && <span className="w-2 h-2 bg-red-500 rounded-full" />}
+      <span className={`w-2 h-2 bg-red-500 rounded-full ${showBadge ? 'visible' : 'invisible'}`} />
```

Comment:
I think we should use block and hidden here, as visible and invisible will always render the element and just not allocate the space vs it is better to not render the element at all if not required

Reply:
Amazing suggestion! Will update!

Reply:
I reverted for now because I couldn't get the desired effect of the icon not moving if the badge comes in. We will have to address that in a separate PR.